test/system/account_management_test.rb

```diff
@@ -1,53 +1,46 @@
 require "application_system_test_case"
 
 class AccountManagementSystemTest < ApplicationSystemTestCase
-  @@test_devices.each do |device_name, display_details|
-    test "user can edit their account and change account settings during registration on a #{device_name}" do
-      resize_for(display_details)
+  device_test "user can edit their account and change account settings during registration" do
+    be_invited_to_sign_up
 
-      be_invited_to_sign_up
+    visit root_path
+    sign_up_from_homepage
 
-      visit root_path
-      sign_up_from_homepage_for(display_details)
+    fill_in "Email", with: "andrew.culver@gmail.com"
+    fill_in "Set Password", with: example_password
+    fill_in "Confirm Password", with: example_password
+    click_on "Sign Up"
 
-      fill_in "Email", with: "andrew.culver@gmail.com"
-      fill_in "Set Password", with: example_password
-      fill_in "Confirm Password", with: example_password
-      click_on "Sign Up"
-
-      if billing_enabled?
-        complete_pricing_page
-      end
-
-      # we should now be on an onboarding step.
-      assert page.has_content?("Tell us about you")
-      fill_in "First Name", with: "Testy"
-      fill_in "Last Name", with: "McTesterson"
-      fill_in "Your Team Name", with: "The Testing Team"
-      page.select "Brisbane", from: "Your Time Zone"
-      click_on "Next"
-
-      assert page.has_content?("The Testing Team’s Dashboard")
-
-      user = User.find_by(email: "andrew.culver@gmail.com")
+    if billing_enabled?
+      complete_pricing_page
+    end
 
-      visit edit_account_user_path(user)
+    # we should now be on an onboarding step.
+    assert_text "Tell us about you"
+    fill_in "First Name", with: "Testy"
+    fill_in "Last Name", with: "McTesterson"
+    fill_in "Your Team Name", with: "The Testing Team"
+    page.select "Brisbane", from: "Your Time Zone"
+    click_on "Next"
 
-      fill_in "Email", with: "andrew.culver.new@gmail.com"
-      fill_in "First Name", with: "Testy.new"
-      fill_in "Last Name", with: "McTesterson.new"
-      page.select "Tokyo", from: "Your Time Zone"
+    assert_text "The Testing Team’s Dashboard"
```

Comment:
In the two sample cases I've replaced `assert page.has_content?(…)` with the equivalent `assert_text` since the latter is easier to read, and helps demonstrate better testing practices to Bullet Train apps.